src/App.css

```
@@ -62,13 +62,44 @@
   color: #162162;
 }
 
-.editor-wrapper {
+.right-wrapper {
   margin: 10px 10px 10px 5px;
   background-color: #e8e8e8;
   flex: 2;
+  max-width: 34%;
```

Absolutely minor, but you might want to change this to:

```suggestion
  max-width: calc(100% / 3);
```